### DISCUSSION TITLE
More Ideas for OpenHV #2

### DISCUSSION THREAD

**Discussion by WilliamWeberBerrutti** [OpenHV/OpenHV#878] [Ideas]
- What if it's possible to build water units and buildings on acid tiles? And, to make it interesting, what if acid reduced the movement speed of the units (by 25% or 50% maybe)? Maybe it also affects unit production speed?
- What if Yuruki's Shield power also affected buildings? What if the only cell that affects buildings is the central one, affecting at most 1 building with it?
- What about reducing the radius of Synapol's Helicopters? Meaning they can group closer to each other?

**Comment by Mailaender** [comment]
There are no acid tiles, it is swamp.

**Comment by SiegeSpud** [comment]
I just use them as acid in recent maps but they are really swamp tiles. See the Green Greens map - that's the intended use. I'm not convinced about buildings on swamp, but I would be willing to suggest more water buildings (on the other side we have to many units and buildings as for now IMHO).
